docs/docs/spec.md

```diff
@@ -209,13 +212,13 @@ For all named annotations referred to by this spec:
 All annotations have runtime retention. None of the annotations are marked
 [repeatable].
 
-## The type-use annotation
+## The type-use annotations
 
-We provide a parameterless type-use annotation called `@Nullable`.
+We provide two parameterless type-use annotations, `@Nullable` and `@NonNull`.
 
 ### Recognized locations for type-use annotations
 
-A location is a *recognized* location for our type-use annotation in the
+A location is a *recognized* location for our type-use annotations in the
```

Comment:
Should you update the non-normative comment below as well?

Reply:
Sorry, in what way?

(Maybe this thread will end up being obsolete now that your #592 removes most of that comment.)

Reply:
I mean this part:

> For now, we've chosen to restrict ourselves to API locations for which tools
> mostly agree on what it means for a type in that location to be `@Nullable`.

What about the other one?

Happy for this to go away if it does.

Reply:
Ah, right. Since this is going in after your PR, we should be good to go.